packages/expo-maps/ios/AppleMapsView.swift

```diff
@@ -159,4 +198,43 @@ struct AppleMapsView: View, AppleMapsViewProtocol {
       }
     }
   }
+
+  private func handleSelectionChange(_ newSelection: MapSelection<MKMapItem>?) {
+    guard let item = newSelection?.value else {
+      return
+    }
+
+    if let marker = props.markers.first(where: { $0.mapItem == item }) {
+      props.onMarkerClick([
+        "title": marker.title,
+        "tintColor": marker.tintColor,
+        "systemImage": marker.systemImage,
+        "coordinates": [
+          "latitude": marker.coordinates.latitude,
+          "longitude": marker.coordinates.longitude
+        ],
+      ])
+      return
+    }
+  }
+
+  private func polyline(
+    at tap: CLLocationCoordinate2D,
+    threshold: CLLocationDistance = 20
```

Comment:
since apple maps doesnt support tapping on polylines by default, this is a custom solution to allow tapping on them. threshold is arbitrary here, perhaps it should be configured by the user? kinda like `hitSlop` for touchables

Reply:
Let's add an option for the user to configure it. We don't want to hide it.